### PR TITLE
Remove hardcoded nodes

### PR DESCRIPTION
**Pull request by AnthonyLaw**
### Add
- request node list (limit on 30) from statistics service by on network type.
- random assign node

### Removed
- hardcoded node list config

It can't be tested until we have the latest statistics service up.

### CODE REVIEW

File: src/services/NetworkService.js [hunk 1]
```diff
@@ -103,4 +107,41 @@ export default class NetworkService {
...
+     */
+    static async getSelectorNodeList(networkType: AppNetworkType) {
+        const nodes = await NetworkService.getNodeList(networkType, 'preferred');
+        let nodeUrls = [];
+
+        for (const node of nodes) {
+            const { apiStatus, host } = node;
+            if (apiStatus) {
+                const isHttps = apiStatus?.isHttpsEnabled || false;
+                nodeUrls.push(`${isHttps ? 'https' : 'http' }://${host}:${isHttps ? '3001' : '3000' }`)
```
[the remark this answers is not in the document]
Comment (cryptoBeliever) [reply src/services/NetworkService.js:151]: I think statistic service should have property like for example `apiUrl` where you have entire URL (protocol + domain + port). So for example https://super.node:3001 or http://super.node:3000 depends if SSL supported or not.

File: src/store/network.js [hunk 2]
```diff
@@ -62,30 +64,65 @@ export default {
...
-            });
-            GlobalListener.setNetwork(network);
             await dispatchAction({ type: 'wallet/initState' });
         },
+        loadNodeList: async ({ commit }) => {
+            try {
+                // load nodes list from statistic service
+                const [testnetNodes, mainnetNodes] = await Promise.all([
+                    NetworkService.getSelectorNodeList('testnet'),
+                    NetworkService.getSelectorNodeList('mainnet')
```
Comment (bassemmagdy) [inline src/store/network.js:116]: why don't we fetch nodes based on network type?

Comment (cryptoBeliever) [reply src/store/network.js:116]: In mobile you have two list of nodes when you login. And you can switch from testnet to mainnet. Not sure if this answers your question :)

Comment (AnthonyLaw) [reply src/store/network.js:116]: The node Selector tab is showing mainnet and testnet list.
IMO we should fetch both.

**Review by cryptoBeliever — APPROVED** [review]

**Review by bassemmagdy — APPROVED** [review]

**Pull request merged**